### ISSUE TITLE
Deserialization exception for age rating: "R+ - Mild Nudity"

### ISSUE THREAD

**Issue by fbaierl** [SandroHc/reactive-jikan#1]
I am currently playing around with your library and I noticed an **InvalidFormatException** concerning the age rating:

```
E/JikanWebService: Unable to load anime with id: 40436
    reactor.core.Exceptions$ReactiveException: net.sandrohc.jikan.exception.JikanResponseException: Error parsing JSON for query: net.sandrohc.jikan.query.anime.AnimeQuery
        at reactor.core.Exceptions.propagate(Exceptions.java:393)
        at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:97)
        at reactor.core.publisher.Mono.block(Mono.java:1680)
        at florian.baierl.daily_anime_news.web.JikanWebService.getAnime(JikanWebService.java:43)
        at florian.baierl.daily_anime_news.repository.JikanRepository.loadAnimeFromWeb(JikanRepository.java:38)
        at florian.baierl.daily_anime_news.ui.animeseasons.JikanViewModel.lambda$fetchAnime$2$JikanViewModel(JikanViewModel.java:61)
        at florian.baierl.daily_anime_news.ui.animeseasons.-$$Lambda$JikanViewModel$2oCfGuNrPuSaa5gfN3aWCnc_GmE.call(Unknown Source:4)
        at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
        at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
        at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
        at io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:614)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)
    	Suppressed: java.lang.Exception: #block terminated with an error
        at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:99)
        		... 16 more
     Caused by: net.sandrohc.jikan.exception.JikanResponseException: Error parsing JSON for query: net.sandrohc.jikan.query.anime.AnimeQuery
        at net.sandrohc.jikan.Jikan.dumpStacktrace(Jikan.java:141)
        at net.sandrohc.jikan.query.Query.deserialize(Query.java:43)
        at net.sandrohc.jikan.-$$Lambda$Jchf727RNBPFlMuul1IpRuaem3I.apply(Unknown Source:4)
        at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:118)
        at reactor.core.publisher.SerializedSubscriber.onNext(SerializedSubscriber.java:99)
        at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.onNext(FluxRetryWhen.java:162)
        at reactor.core.publisher.Operators$MonoSubscriber.complete(Operators.java:1782)
        at reactor.core.publisher.MonoFlatMap$FlatMapInner.onNext(MonoFlatMap.java:241)
        at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onNext(FluxDoFinally.java:123)
        at reactor.core.publisher.FluxHandle$HandleSubscriber.onNext(FluxHandle.java:112)
        at reactor.core.publisher.FluxMap$MapConditionalSubscriber.onNext(FluxMap.java:213)
        at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onNext(FluxDoFinally.java:123)
        at reactor.core.publisher.FluxHandleFuseable$HandleFuseableSubscriber.onNext(FluxHandleFuseable.java:178)
        at reactor.core.publisher.FluxContextStart$ContextStartSubscriber.onNext(FluxContextStart.java:96)
        at reactor.core.publisher.Operators$MonoSubscriber.complete(Operators.java:1782)
        at reactor.core.publisher.MonoCollectList$MonoCollectListSubscriber.onComplete(MonoCollectList.java:121)
        at reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:252)
        at reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:136)
        at reactor.netty.channel.FluxReceive.onInboundComplete(FluxReceive.java:378)
        at reactor.netty.channel.ChannelOperations.onInboundComplete(ChannelOperations.java:373)
        at reactor.netty.channel.ChannelOperations.terminate(ChannelOperations.java:429)
        at reactor.netty.http.client.HttpClientOperations.onInboundNext(HttpClientOperations.java:625)
E/JikanWebService:     at reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:96)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
        at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
        at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1526)
        at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1275)
        at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1322)
        at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:501)
        at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:440)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
        at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        	... 1 more
     Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `net.sandrohc.jikan.model.enums.AgeRating` from String "R+ - Mild Nudity": not one of the values accepted for Enum class: [R+ - Mild Nudity (may also contain violence & profanity), Rx, G, Rx - Hentai (extreme sexual content/nudity), R, PG-13 - Teens 13 or older, R - 17+ (violence & profanity), PG, PG-13, R - 17+ recommended (violence & profanity), PG - Children, G - All Ages, R+]
        at [Source: (byte[])"{"request_hash":"request:anime:2d20731b3c87c364862b1fe36852cdddc5f9c91a","request_cached":true,"request_cache_expiry":13338,"mal_id":40436,"url":"https:\/\/myanimelist.net\/anime\/40436\/Peter_Grill_to_Kenja_no_Jikan","image_url":"https:\/\/cdn.myanimelist.net\/images\/anime\/1368\/108441.jpg","trailer_url":null,"title":"Peter Grill to Kenja no Jikan","title_english":"Peter Grill and the Philosopher's Time","title_japanese":"\u30d4\u30fc\u30bf\u30fc\u30fb\u30b0\u30ea\u30eb\u3068\u8ce2\u8005\u306"[truncated 1936 bytes]; line: 1, column: 862] (through reference chain: net.sandrohc.jikan.model.anime.Anime["rating"])
        at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:1702)
        at com.fasterxml.jackson.databind.DeserializationContext.handleWeirdStringValue(DeserializationContext.java:947)
        at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._deserializeAltString(EnumDeserializer.java:255)
        at com.fasterxml.jackson.databind.deser.std.EnumDeserializer.deserialize(EnumDeserializer.java:179)
        at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:293)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:156)
        at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4482)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3495)
        at net.sandrohc.jikan.query.Query.deserialize(Query.java:41)
        	... 53 more`
```

**Comment by fbaierl** [comment]
I have opened a pull request here: https://github.com/SandroHc/reactive-jikan/pull/2

**Comment by SandroHc** [comment]
Fixed by #2. Once again, thank you for your help. 🥇
